bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -88,7 +89,23 @@ public void onFinished() {
                 logger.debug("Scan for binding '{}' successfully finished.", bindingId);
             }
         });
-        return Response.ok().build();
+        
+        return Response.ok(new ScanResponseResultDto(discoveryServiceRegistry.getMaxScanTimeout(bindingId))).build();
+    }
+    
+    /**
+     * Dummy class needed to create a proper JSON formating of the response
+     * 
+     * @author ivaylo.ivanov
+     *
+     */
+    @SuppressWarnings("unused")
+    private static final class ScanResponseResultDto {
```

Comment:
Yes, we do. 

I tried simply returning the integer, but then - the response was empty.